app/config/config.yml

```diff
@@ -54,6 +54,9 @@ twig:
     strict_variables: "%kernel.debug%"
     form_themes:
         - 'PrestaShopBundle:Admin/TwigTemplateForm:bootstrap_3_horizontal_layout.html.twig'
+    globals:
+        configuration: "@prestashop.adapter.legacy.configuration"
```

Comment:
this make tests cry, also I dont think it is a good practice to declare a global in Symfony that rely on legacy class :(